cmds/exp/netbootxyz/network.go

```diff
@@ -69,15 +69,24 @@ func closeIO(c io.Closer, err *error) {
 }
 
 func configureDHCPNetwork() error {
-	if *verbose {
-		log.Printf("Trying to configure network configuration dynamically...")
-	}
-
 	link, err := findNetworkInterface(*ifName)
 	if err != nil {
 		return err
 	}
 
+	addrs, err := netlink.AddrList(link, netlink.FAMILY_ALL)
+	if err != nil {
+		return fmt.Errorf("Can't enumerate addresses: %v", err)
+	}
+	if len(addrs) > 0 {
```

Comment:
If this check is enough, we don't really need to add the `-no-dhcp` flag, I think.

Reply:
Yes, this check looks sufficient.